iOS Client/DevotionsController/DevotionNotification.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  DevotionNotification.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 11/30/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+
+import Foundation
+import UserNotifications
+
+class DevotionNotificationContent: UNMutableNotificationContent {
+    var devotion: MBDevotion
+    
+    init(devotion: MBDevotion) {
+        self.devotion = devotion
+        super.init()
+        self.title = devotion.verse
+        self.body = "Read your daily devotion from Mockingbird"
+        self.sound = UNNotificationSound.default()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
```

Comment:
Hmm, wonder what the deal with this is? It breaks if we delete it?

Reply:
the swift docs are kind of unclear. They say all subclasses have to provide an implementation or required initializers but also that `You do not have to provide an explicit implementation of a required initializer if you can satisfy the requirement with an inherited initializer.` I don't know what that means . . .

Reply:
Yeah I'm a little confused about this. I don't think this ever actually would get called but it would be nice to not include it